.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Summary
+<!-- Simple summary of what was changed. -->
+
+# Motivation
+<!-- Why are you making this change? If it's for fixing a bug, if possible, please include a code snippet or example project that demonstrates the issue. -->
+
+# Testing
+<!-- How was the code tested? Be as specific as possible. -->
+- [ ] Added tests
+- [ ] Modified tests
+- [ ] Manually verified
+
+# Screenshots
```

Comment:
Do we really need this section? Not every PR has screenshots and they could be added inside the Summary

Reply:
This is the template we used for native SDKs, we just leave them empty for PRs without screenshots